src/components/load_from_csv/fondant_component.yaml

```diff
@@ -0,0 +1,29 @@
+name: Load from csv file
+description: Component that loads a dataset from huggingface hub
+image: hakimovich99/load_from_csv:dev #TODO: change it to fndnt once implemented
```

Comment:
You can use the `ml6team` github container registry for any components related to a use case.

Reply:
Just to make sure, like that: `ghcr.io/ml6team/load_from_csv:dev` ?

Reply:
Yes. You can label them to link them to this repo. 
See [this script](https://github.com/ml6team/fondant-usecase-datacomp/blob/main/scripts/build_components.sh) for instance, it uses the `fondant build` command.

Reply:
Thanks!